blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -67,15 +92,15 @@ Blockly.ContractEditor.EXAMPLE_BLOCK_TYPE = 'functional_example';
 Blockly.ContractEditor.EXAMPLE_BLOCK_ACTUAL_INPUT_NAME = 'ACTUAL';
 
 Blockly.ContractEditor.typesToColorsHSV = {
-  'none': [0, 0, 0.6],
+  // 'none': [0, 0, 0.6], // not available to users
   'Number': [192, 1.00, 0.99], // 00ccff
   'string': [180, 1.00, 0.60], // 0099999
   'image': [285, 1.00, 0.80], // 9900cc
   'boolean': [90, 1.00, 0.4] // 336600
 };
```

Comment:
This is duplicated in at least one other place (functionalBlockUtils). I wonder if we can/should get it in just one place.